runtime/src/asset.rs

```diff
@@ -605,6 +648,7 @@ decl_module! {
             ensure!(investor_dids.len() == values.len(), "Investor/amount list length inconsistent");
             ticker.canonize();
             ensure!(Self::is_owner(&ticker, did), "user is not authorized");
+            ensure!(!Self::frozen(&ticker), "token is frozen");
```

Comment:
If I remove that check, `batch_issue` will succeed and return `Ok(())` with empty `investor_dids` and `values`. Is this acceptable for you, @satyamakgec?

Reply:
I don't think so because if you have checked in `_is_valid_transfer()` then it will fail the transaction. As batch_issue calls `_is_valid_transfer()` functions in every iteration of issuance.

Reply:
There are no iterations when the vectors are empty. `batch_issue` succeeds vacuously.

Reply:
Then it is a bug in the batch_issue function it also checks `investor_dids.len() > 0`

Reply:
It definitely returns `Ok(())` in the unit test in this PR when the vectors are empty.

Reply:
There is no `>` check in this function. Only a check the the length of the two vector arguments is the same. I'll add a `>` check.